### ISSUE TITLE
Trier les protocoles sur la page d'accueil de monitoring ?

### ISSUE THREAD

**Issue by Splendens** [PnX-SI/gn_module_monitoring#164]
Bonjour, 

Est-ce qu'il possible de trier les protocoles sur la page d'accueil de monitoring ?
Ils ne semblent être triés ni par id_module, ni par ordre alphabétique ou date, et le module_order n'est pas pris en compte.

**Comment by maximetoma** [comment]
Effectivement, pour ma part, je commence aussi à avoir pas mal de sous-modules et sur ma page d’accueil admin, ça commence à devenir un sacré bazar :)

J'ai résolu en partie le problème en rajoutant une méthode "order_by" dans le fichier `./gn_module_monitoring/backend/modules/repositories.py` correspondant à l'API `geonature/api//monitoring/modules` qui gère la récupération des modules

En gros, il faut modifier la section lignes 88 à 94 en rajoutant `.order_by(TMonitoringModules.<nomdelacolonne>)` avant le .all()

        try:
            res = (
                DB.session.query(TMonitoringModules).order_by(TMonitoringModules.<nomdelacolonne>)
                .all()
            )

            return res

Les colonnes dispo sont normalement celles présentes dans l'API mais je n'ai pas fait de tests pour savoir si toutes les colonnes marchaient. Personnellement, j'ai demandé un ordre par la colonne `module_code` et ça fonctionne

Or ce n'est pas très conventionnel de modifier ça comme ça... L'idéal serait de le paramétrer dans un gn_module_config.toml mais je ne sais pas trop comment faire....

**Comment by camillemonchicourt** [comment]
Salut, je ne pense que cela mérite un paramètre.
Mais en effet trier par défaut sur le label du module (plutôt que son code) serait une bonne chose.

Pour aller plus loin dans le rangement, il sera pertinent de pouvoir grouper les sous-modules par blocs de protocoles : https://github.com/PnX-SI/gn_module_monitoring/issues/80
